scripts/motor_driver.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+import rospy
+import rosparam
+
+from geometry_msgs.msg import Twist
+from std_msgs.msg import Float64, Bool
+
+import numpy as np
+
+from dual_max14870_rpi import DualMotorDriver
+
+try:
+    pin = rospy.get_param('driver_pinout')
+    rospy.logwarn("no 'driver_pinout' param found using default config file")
+except:
+    try:
+        (pin, namespace) =  rosparam.load_file("./config/driver_L_pinout.yaml", )
+        rospy.logwarn("no default config file using hard coded pinout")
+    except:
+        pin ={ 'FAULT':23, 'PWM2':24, 'PWM1':12, 'DIR2':16, 'DIR1':20, 'EN': 21}
```

Comment:
This is a bit exaggerated, better to pass the default parameters and get the parameters using as namespace the node's name.
```suggestion
pin = rospy.get_param(rospy.get_name(), {'FAULT':23, 'PWM2':24, 'PWM1':12, 'DIR2':16, 'DIR1':20, 'EN': 21})
```